### PR TITLE
Add tests for flipProbabilityDp

### PR DESCRIPTION
**Pull request by csharrison**

### CODE REVIEW

**Comment by csharrison** [comment]
Let me know if you have concerns about `deepStrictEquals` comparisons for floats.

File: flexible-event/privacy.test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        epsilon: 14,
+        expected: 0.000002494582008677539
+    },
+]
+
+flipProbabilityTests.forEach(test => {
+    const flipProb = flipProbabilityDp(test.numStates, test.epsilon);
+    assert.deepStrictEqual(flipProb, test.expected);
+})
+
```
Comment (apasel422) [inline flexible-event/privacy.test.ts:31]: nit: Fix the line ending here.

**Review by apasel422 — CHANGES_REQUESTED** [review]
Could we also set up a workflow like https://github.com/WICG/attribution-reporting-api/blob/main/.github/workflows/test-header-validator.yml for automatic testing on changes?

**Comment by csharrison** [comment]
> Could we also set up a workflow like https://github.com/WICG/attribution-reporting-api/blob/main/.github/workflows/test-header-validator.yml for automatic testing on changes?

Yep, done

**Comment by csharrison** [comment]
Oops let me debug the failing build

**Review by apasel422 — APPROVED** [review]

**Pull request merged**